src/components/network/network-loader.js

```diff
@@ -168,7 +170,7 @@ genotet.NetworkLoader.prototype.addGenes_ = function(genes) {
   });
 
   if (!newGenes.length) {
-    genotet.warning('genes are already in the network');
+    genotet.warning('genes are already in the network or not existing');
```

Comment:
Can we check for each case and display "already in the network" and "not existing" separately?